microsoft-edge/webview2/concepts/process-related-events.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Handling Process-related Events in WebView2
+description: Process-related events in WebView2 and how apps can handle them for recovery and improved reliability.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 06/20/2023
+---
+# Handling Process-related Events in WebView2
+## Overview
+The WebView2 API provides the `CoreWebView2.ProcessFailed` and `CoreWebView2Environment.BrowserProcessExited` events for your application to react to different scenarios. Use this document to learn how to use these events to react when these scenarios present.
+
+  * `CoreWebView2.ProcessFailed`. Use this event for diagnostics and recovery from failures in the WebView2 processes.
+  * `CoreWebView2Environment.BrowserProcessExited`. Use this event to synchronize operations involving WebView2 Runtime resources and lifetime, such as User Data Folder management and updates.
+
+To improve the reliability of your WebView2 application, it is recommended that it handles at least the following events:
+  * [The main browser process has exited unexpectedly](<link>).
+  * [A renderer process has exited unexpectedly](<link>).
+  * [A renderer process has become unresponsive](<link>).
+
+> [!NOTE]
+> This document is a high-level overview of the aforementioned events. Ultimately, the reference documentation for each individual API mentioned in this document shall prevail.
+
+## WebView2 Process Model and related APIs
+The [WebView2 Process Model](process-model.md) uses the WebView2 Runtime to support your WebView2 application. Because most of the magic in the WebView2 controls in your application happens out-of-process (in the WebView2 Runtime), failures and general information about these processes are passed to your application through different WebView2 APIs. The APIs used to report this information to your application include:
+
+  * CoreWebView2
+    - `BrowserProcessId` property
+    - `ProcessFailed` event
+  * CoreWebView2Environment
+    - `BrowserProcessExited` event
+    - `ProcessInfosChanged` event
+    - `GetProcessInfos` method
```

Comment:
```suggestion
```
replaced by tab-set below.  entered as a separate comment in case you want to temporarily retain the present easy-to-read list for comparison to the tab-set.